computer-vision.cfg

```diff
@@ -1,2362 +0,0 @@
-# Doxyfile 1.8.9.1
```

Comment:
How can I use the documentation generator to write description into header files and source files?